nextflow_schema.json

```diff
@@ -207,17 +199,40 @@
         {
             "$ref": "#/definitions/input_output_options"
         },
-        {
-            "$ref": "#/definitions/reference_genome_options"
-        },
-        {
-            "$ref": "#/definitions/institutional_config_options"
-        },
         {
             "$ref": "#/definitions/max_job_request_options"
         },
         {
             "$ref": "#/definitions/generic_options"
         }
-    ]
+    ],
+    "properties": {
```

Comment:
what are these?

Reply:
I think this happened when I made changes in the schema editor? uncertain